request/send_notification.php

```diff
@@ -0,0 +1,63 @@
+<?php
+    require_once(dirname(__FILE__).'/../includes/application.php');
+
+    $output = array(
+        'response' => false,
+        'message' => 'Something went wrong'
+    );
+    if (!empty($UserObj)) {
+        if (isset($_POST) && isset($_POST['type'])) {
+            switch ($_POST['type']) {
+                case 'send_sms_circle_of_trust':
+                    $types = array(
+                        'come_get_me',
+                        'need_interruption',
+                        'need_to_talk'
+                    );
+                    if (!empty($_POST['msg_type']) &&
+                        in_array($_POST['msg_type'], $types)
+                    ) {
+                        $msg_type = $_POST['msg_type'];
+                        $message = '';
+                        switch ($msg_type) {
+                            case $types[0]:
+                                $message = "Come and get me.I need help getting home safely.
+                                Call ASAP to get my Location.
+                                via FirstAide web app";
```

Comment:
@MehaKaushik They were provided in the requirements, and Akanksha used them last year too.